ddtelemetry/src/worker/mod.rs

```diff
@@ -636,6 +636,7 @@ impl TelemetryWorker {
             runtime_id: &self.runtime_id,
             seq_id,
             host: &self.data.host,
+            origin: "unknown",
             application: &self.data.app,
```

Comment:
Could you leave it empty if it's not available?

Reply:
Done